src/lib/es2015.symbol.wellknown.d.ts

```diff
@@ -221,7 +221,9 @@ interface String {
     /**
      * Replaces text in a string, using an object that supports replacement within a string.
      * @param searchValue A object can search for and replace matches within a string.

```

Comment:
Looking at it again, I think the additional sentence should actually be here. Also the wording of this line is pretty poor, and could be improved.

Reply:
Will update shortly to maybe as minimal as possible.

Reply:
It should be good now? 
This is what I put:

> Replaces first match with string or all matches with RegExp.
      @param searchValue A string or RegExp search value.
      @param replaceValue A string containing the text to replace for match.